pkg/controllers/termination/controller.go

```diff
@@ -46,27 +47,27 @@ import (
 const controllerName = "termination"
 
 var (
-	terminationSummaryVec = prometheus.NewSummaryVec(
+	terminationSummary = prometheus.NewSummary(
```

Comment:
Does this change up the underlying data at all? What's the difference between NewSummary and NewSummaryVec?

Reply:
Loooks good to me here.